admin-dev/themes/new-theme/js/pages/product/components/categories/tags-renderer.ts

```diff
@@ -96,15 +101,19 @@ export default class TagsRenderer {
   }
 
   private listenTagRemoval(): void {
+    if (!this.container) {
+      console.error('The container is undefined or invalid');
```

Comment:
You're doing a little more things that what you described in the PR description 😄

Reply:
Mmm, yep ^^"
Is it bad?